File: web2-front-app/src/Models/Product.js
export class Product {
    constructor(
      ProductId=0,
      Name="",
      
      Description="",
      Price=0,
      Quantity=1,
      Image="",
      SellerId=0,
      Category,
      
    ) {
      this.productId=ProductId;
      this.name=Name;
      this.description=Description;
      this.price=Price;
      this.quantity=Quantity;
      this.image=Image;
      this.sellerId = SellerId;
      this.category=Category;
      this.sellerName="";
    }
  }
  export default Product;
  
  

File: Services/Models/Product.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Services.Models
{
    public class Product
    {
        public int ProductId { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public double Price { get; set; }
        public int Quantity { get; set; }
        public string Image { get; set; }
        //public User Seller { get; set; }
        public int SellerId { get; set; }
        public List<OrderProduct> Orders { get; set; }
        public string Category { get; set; }
    }
}
